recipe/meta.yaml

```diff
@@ -23,11 +25,15 @@ requirements:
     - glib 2.51.*
     - freetype 2.7|2.7.*
     - icu 58.*
+    - graphite2 1.3.*
+    - gobject-introspection 1.51.*
+    - python >=2.7,<3
```

Comment:
The ` >=2.7,<3` is better at the `skip` session than here. `skip: True  # [py3k or win]` should do it.